### PR TITLE
Bug 1136902 - Added test file and method for utils sanitize method. r…

### PR DESCRIPTION
**Pull request by sr-murthy**
…=sandeep@sandeepmurthy.is

Bug 1136902 - Fixed PEP8 linter errors in last commit. r=sandeep@sandeepmurthy.is

Bug 1136902 - Separate sanitize test into separate resource type sanitize tests

Bug 1136902 - Move cookies file into cookies subfolder

Bug 1136902 - Remove old cookies file

Bug 1136902 - Remove trailing whitespace

Bug 1136902 - requested changes from last review

Bug 1136902 - removed line break

Bug 1136902 - Fixed test failures, plus requested changes from last review

### CODE REVIEW

**Comment by sr-murthy** [comment]
Sorry, to create a new PR, but I tried to forcibly update the old PR and it just was not working for some reason.

File: firefox_puppeteer/tests/test_utils.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from firefox_ui_harness import FirefoxTestCase
```
Comment (whimboo) [inline firefox_puppeteer/tests/test_utils.py:5]: This won't work anymore. The testcase class is no longer part of the harness but puppeteer.  See all the other tests. The reason why Travis did not fail is that you haven't added the new file to the manifest. So best practice is always to check if your working branch has all the latest changes from the master/mozilla-central branch. 

Let me do the changes and push it, given that it somewhat blocks my work on bug 1232967.

**Comment by whimboo** [comment]
I got this PR merged to mozilla-central as https://github.com/mozilla/firefox-ui-tests/commit/8330ae2bbbf945660181cfe2de3b56bf30850fce with the before-mentioned changes added.

**Comment by sr-murthy** [comment]
Thank you.

**Pull request closed without merging**